src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -229,20 +229,51 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
         }
     }
 
+    private isChildAssetType = (asset: IAsset): boolean => {
```

Comment:
the naming of this method is confusing.  As discussed, we should call it `isContainerType` or something

Reply:
Renamed to `isTaggableAssetType`.